README.md

```diff
@@ -18,10 +18,40 @@ We would love involvement from more people! If you notice any errors or would li
 
 We use GitHub issues for [tracking bugs and feature requests](https://github.com/datatogether/website/issues) and Pull Requests (PRs) for [submitting changes](https://github.com/datatogether/website/pulls)
 
+## Usage
+
+
 ## Development
 
-1. Get [hugo](https://github.com/gethugoio/hugo)
-2. `git clone` this repo & `cd` into le directory
-3. `hugo server`
-4. run a raw `hugo` before running `git commit`
-5. :heart:
+1. Clone this repo
+
+  ```
+  $ git clone git@github.com:datatogether/website.git
+  ```
+
+  and `cd` into the directory
+
+1. Install Hugo
```

Comment:
In the formatted version, this number doesn't get incremented to 2 automatically (probably because of the intervening paragraph). Suggest either indenting the paragraph or manually numbering this to 2.